### ISSUE TITLE
Button Font Colour Same as Button Colour

### ISSUE THREAD

**Issue by FalconXray** [worldanvil/worldanvil-bug-tracker#975]
**World Anvil Username**: FalconX

**World / Campaign / RPG Character / Manuscript Title**: Character Sheet/Relationships/Create new article under Bio Parents

**Feature**: 

**Related URLS** (To public & visible pages add screenshots otherwise):

https://www.worldanvil.com/world/person/15cc7a73-eed9-44f2-9b12-ae7009e42609/edit

**Describe the Issue**
A clear and concise description of what the issue is.

The button to add the article and save the info appears with no text on it and at first, I thought it was just a box and lost 
the info I typed in.  On mouse over the text appears in a slight grey font

**To Reproduce**

**Expected behavior**
Button that is visible

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!![Screen Shot 2022-04-10 at 8 10 31 pm](https://user-images.githubusercontent.com/102790891/162613370-a6e9940e-908c-45bc-841a-f094aecd6c5c.png)

**Comment by iamromeo** [comment]
I am sorry @FalconXray but which browser are you using. I cannot replicate this on any of the browsers we are supporting

**Comment by iamromeo** [comment]
@Kordishal can you ?

**Comment by Kordishal** [comment]
I cannot reproduce this either no.

Can you test if this happens in a private / incognito window?
In a different browser?
On a different device?

And if you know how, check what kind of CSS is affecting this element that causes this issue would be the most helpful to track it down.

**Comment by iamromeo** [comment]
Yes Safari we do support but I am checking my Safari and I cannot replicate it.

**Comment by FalconXray** [comment]
I will try again to reproduce this and take careful note of the steps I travel.

**Comment by FalconXray** [comment]
appearance: button;
background-attachment: scroll;
background-clip: border-box;
background-color: rgb(59, 146, 187);
background-image: none;
background-origin: padding-box;
background-position-x: 0%;
background-position-y: 0%;
background-size: auto;
border-bottom-color: rgb(255, 255, 255);
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
border-bottom-style: none;
border-bottom-width: 0px;
border-image-outset: 0;
border-image-repeat: stretch;
border-image-slice: 100%;
border-image-source: none;
border-image-width: 1;
border-left-color: rgb(59, 146, 187);
border-left-style: solid;
border-left-width: 1px;
border-right-color: rgb(59, 146, 187);
border-right-style: solid;
border-right-width: 1px;
border-top-color: rgb(255, 255, 255);
border-top-left-radius: 3px;
border-top-right-radius: 3px;
border-top-style: none;
border-top-width: 0px;
box-sizing: border-box;
color: rgb(255, 255, 255);
cursor: pointer;
display: block;
font-family: "open sans", "helvetica neue", Helvetica, Arial, sans-serif;
font-size: 12.800000190734863px;
font-weight: 400;
height: 27px;
line-height: 19.200000762939453px;
padding-bottom: 4px;
padding-left: 8px;
padding-right: 8px;
padding-top: 4px;
text-align: center;
text-decoration: none;
transition-delay: 0s, 0s, 0s, 0s;
transition-duration: 0.15s, 0.15s, 0.15s, 0.15s;
transition-property: color, background-color, border-color, box-shadow;
transition-timing-function: ease-in-out, ease-in-out, ease-in-out, ease-in-out;
vertical-align: middle;
width: 200.875px;
UngroupedVariables
--blue: #007bff;
--borderWidths-0: ;
--breakpoint-lg: 992px;
--breakpoint-md: 768px;
--breakpoint-sm: 576px;
--breakpoint-xl: 1200px;
--breakpoint-xs: 0;
--colors-bounds: rgba(65, 132, 244, 1);
--colors-boundsBg: rgba(65, 132, 244, 0.05);
--colors-hover: #ececec;
--colors-overlay: rgba(0, 0, 0, 0.15);
--colors-overlayContrast: rgba(255, 255, 255, 0.15);
--colors-panel: #fefefe;
--colors-panelContrast: #ffffff;
--colors-selected: rgba(66, 133, 244, 1);
--colors-selectedContrast: #fefefe;
--colors-sponsor: #ec6cb9;
--colors-sponsorContrast: #ec6cb944;
--colors-text: #333333;
--colors-tooltip: #1d1d1d;
--colors-tooltipContrast: #ffffff;
--colors-warn: rgba(255, 100, 100, 1);
--cyan: #17a2b8;
--danger: #dc3545;
--dark: #343a40;
--fa-font-brands: normal 400 1em/1 "Font Awesome 6 Brands";
--fa-font-duotone: normal 900 1em/1 "Font Awesome 6 Duotone";
--fa-font-light: normal 300 1em/1 "Font Awesome 6 Pro";
--fa-font-regular: normal 400 1em/1 "Font Awesome 6 Pro";
--fa-font-solid: normal 900 1em/1 "Font Awesome 6 Pro";
--fa-font-thin: normal 100 1em/1 "Font Awesome 6 Pro";
--font-family-monospace: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
--font-family-sans-serif: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
--fonts-body: "Recursive", system-ui, sans-serif;
--fonts-mono: "Recursive Mono", monospace;
--fonts-ui: "Recursive", system-ui, sans-serif;
--fontSizes-0: 10px;
--fontSizes-1: 12px;
--fontSizes-2: 13px;
--fontSizes-3: 16px;
--fontSizes-4: 18px;
--gray: #6c757d;
--gray-dark: #343a40;
--green: #28a745;
--indigo: #6610f2;
--info: #17a2b8;
--light: #f8f9fa;
--orange: #fd7e14;
--pink: #e83e8c;
--primary: #4f6d88;
--primary-dark: #243d54;
--purple: #6f42c1;
--radii-0: 2px;
--radii-1: 4px;
--radii-2: 8px;
--radii-3: 12px;
--radii-4: 16px;
--red: #dc3545;
--secondary: #6c757d;
--shadows-2: 0px 1px 1px rgba(0, 0, 0, 0.14);
--shadows-3: 0px 2px 3px rgba(0, 0, 0, 0.14);
--shadows-4: 0px 4px 5px -1px rgba(0, 0, 0, 0.14);
--shadows-8: 0px 12px 17px rgba(0, 0, 0, 0.14);
--shadows-12: 0px 12px 17px rgba(0, 0, 0, 0.14);
--shadows-24: 0px 24px 38px rgba(0, 0, 0, 0.14);
--shadows-key: 1px 1px rgba(0,0,0,1);
--shadows-panel: 0px 0px 16px -1px rgba(0, 0, 0, 0.05), 0px 0px 16px -8px rgba(0, 0, 0, 0.05), 0px 0px 16px -12px rgba(0, 0, 0, 0.12), 0px 0px 2px 0px rgba(0, 0, 0, 0.08);
--space-0: 2px;
--space-1: 3px;
--space-2: 4px;
--space-3: 8px;
--space-4: 12px;
--space-5: 16px;
--success: #28a745;
--teal: #20c997;
--warning: #ffc107;
--white: #fff;
--yellow: #ffc107;



Hi Guys,

I switched to dark mode so the contrast would be better.

The Button still is white.

I opened up inspector and highlighted the button to see what css was operating. 

Warm Regards

Lachlan

**Comment by Kordishal** [comment]
And this does happen when you are in a private / incognito window?

**Comment by FalconXray** [comment]
No just in a new window. Not private.

**Comment by Kordishal** [comment]
then this issue is likely caused by an extension on your side. Try disabling them to find the culprit.

**Comment by FalconXray** [comment]
Hi,
I do not have any extensions installed.

**Comment by Kordishal** [comment]
Then I am all out of ideas what could be the issue. We would need the exact property & rule that is causing this. The entire list of just properties does not really help, as we cannot tell from this list which are active.

I will be closing this issue as it is most likely an issue on your system and not something that World Anvil can really fix.

**Comment by FalconXray** [comment]
Thank you for investigating

**Comment by FalconXray** [comment]
Ok I have done a bit of work on this.





This seems to be the list of classes responsible for most of the formatting of the box. So changed them a little so I could see some changes.



So I have copied the classes from the button that creates the the family tree embed.

I was then able to find the variables responsible for the colour of the button.

It is close to the middle of the graphic.  —webkit-appearance: button.  When active the button is clear or white.

When I deactivated the variable this is what happened.


The button is now visible in the —primary. Colour. 

Does this help.

Warm Regards

Lachlan Grierson.
